packages/order-book/src/quoteAmountsAndCosts/README.md

```
@@ -0,0 +1,445 @@
+
+This document explains how CoW Protocol calculates order amounts and costs,
+from the initial `https://api.cow.fi/mainnet/api/v1/quote` API response to the final signed order.
+
+## 1. What `/quote` API returns
+
+The `/quote` endpoint returns [`OrderQuoteResponse`](../generated/models/OrderQuoteResponse.ts)
+which contains [`OrderParameters`](../generated/models/OrderParameters.ts) (in the `quote` field)
+along with additional metadata.
+
+`/quote` API response example:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)